src/m365/teams/commands/user/user-app-remove.ts

```diff
@@ -13,6 +14,7 @@ interface CommandArgs {
 interface Options extends GlobalOptions {
   id: string;
   userId: string;
+  userName: string;
```

Comment:
Since the user can specify either `userName` or `userId` both should be marked as optional